scRNA-seq/04-tag-based_scRNA-seq_processing.Rmd

```diff
@@ -0,0 +1,246 @@
+mkdir -p data/tabula-muris/alevin-quant/10X_P4_3
+```
+
+In this scenario, `10X_P4_3` refers to the sample name that we will be processing this time.
+
+## Step 3: Index the mouse transcriptome with Salmon
+
+Before you can quantify with Salmon and [Alevin](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1670-y) we need our species' [transcriptome to be indexed](https://github.com/AlexsLemonade/training-txome-prep), just was we did with bulk RNA-seq. 
+
+You can use the same transcriptome indexes as bulk RNA-seq, however, due to the shorter read lengths in the 10X sequencing, we will want to use shorter kmers than the default.
+In this instance, we used a `-k` of 23.
+
+In the interest of time, we have already run the command below and have the index built and ready for you in a shared directory.
+
+But for your own reference, here is how you might do it yourself:
+```
+# salmon --threads=16 --no-version-check index \
+#  -t Mus_musculus.GRCm38.cdna.all.fa.gz \
+#  -i index/Mus_musculus/short_index \
+#  -k 23
+```
+
+## Step 4: For each sample, run [Alevin](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1670-y) for quantification
+
+From the command line, running Alevin is not too much different from running
+Salmon for bulk RNA-seq.
+
+### Running Salmon Alevin
+
+Copy and paste this in your `Terminal` to run the Alevin quantification.
+This will take about 20 minutes to run, so we will start now, then talk about the options.
+
+```
+salmon alevin -l ISR \
+  -i /shared/data/reference/refgenie/mm10_cdna/salmon_index/short/short/ \
+  -1 data/tabula-muris/fastq-raw/10X_P4_3/10X_P4_3_L001_R1_001.fastq.gz \
+  -2 data/tabula-muris/fastq-raw/10X_P4_3/10X_P4_3_L001_R2_001.fastq.gz \
+  --chromium  \
+  -p 5 \
+  -o data/tabula-muris/alevin-quant/10X_P4_3 \
+  --tgMap data/tabula-muris/mm_ensdb95_tx2gene.tsv \
+  --dumpFeatures
+```
+
+### Salmon/Alevin Options
+
+You'll recognize a lot of these options as the same as regular `Salmon` such as
+- `-l` to designate library type
+- `-1` and `-2` for file input
+- `-o` to designate a folder for output
+
+Notation for these 10X files is:
+- `R1` files contain the barcodes for cells as well as the UMIs
+- `R2` files contain the full reads for that sample.  
+
+#### `-l`
+As mentioned, `-l` is for designating library type. For all single-cell quant,
+you will want to use the `ISR` library type.
+See [Salmon's documentation](https://salmon.readthedocs.io/en/latest/library_type.html) for more information on fragment library types.
+
+#### `--chromium`
+Because we are using 10X chromium data, we have to use this flag. 
+However, Drop-seq data is also supported, and in this case you would use a `--dropseq` flag instead of this.
+
+#### `--tgMap`
+This is needed to supply a transcript to gene key that Alevin will use to quantify the genes.
+For our example, we've pre-made the file `Mus_musculus.GRCm38_tx2gene.tsv` from the Ensembl transcriptome that we indexed above. 
```

Comment:
If I recall correctly, it _did not like_ this file and it looks like you're using `data/tabula-muris/mm_ensdb95_tx2gene.tsv` below (and that file is being added to the repository in this PR).

Reply:
Correct; I need to fix the file name here.